docs/appendices.txt

```diff
@@ -5,7 +5,7 @@
 [appendix]
 == API Documentation (JavaDoc)
 
-Refer to the JanusGraph API documentation for a complete documentation of all core APIs exposed by JanusGraph.
+Refer to the JanusGraph https://javadoc.io/doc/org.janusgraph/janusgraph-core[API documentation] for a complete documentation of all core APIs exposed by JanusGraph.
```

Comment:
Since this will be compiled with the rest of the docs, what about changing the link here to:

```
https://javadoc.io/doc/org.janusgraph/janusgraph-core/$MAVEN{{project.version}}
```

instead, as that will automatically point to the same version of the API docs as the docs themselves relate to? All other versions are still discoverable from that same page, but the default also matters.